docs/sources/contribute/documentation.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Documentation"
+description = "Contributing to documentation"
+type = "docs"
+[menu.docs]
+parent = "contribute"
+weight = 2
++++
+
+# Contributing to documentation
+
+## How do I contribute?
+
+If you’re unsure about where to start, check out some of our [open docs issues](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3Atype%2Fdocs).
+
+Sometimes it can be difficult to understand an issue when you're just getting started. We strive to keep a collection of [beginner-friendly issues](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3Atype%2Fdocs+label%3A"beginner+friendly") that is more suitable for first-time contributors.
+
+When you’ve found an issue you want to work on, you’re encouraged to comment on the issue to let other people know you intend to work on it.
```

Comment:
@marcusolsson What about linking David's talk here (https://www.youtube.com/watch?v=qPVrkv_s_Bc&list=PLDGkOdUX1UjqKc3ryyoSpWZvs7yktklQr&index=35) Not 100% sure docs is the right place to add that.

Reply:
@dprokop Thanks for the link! I'm thinking that would go perfect on a general contributing docs. Maybe https://grafana.com/docs/contribute. I'll make sure to save it for when we get there!

Reply:
Yes! Marked this as off-topic when I released this is docs contrib readme :)